GitUI/UserControls/FileStatusList.cs

```diff
@@ -400,6 +400,19 @@ public IEnumerable<GitItemStatusWithParent> SelectedItemsWithParent
                 .Where(i => i.Group?.Tag is GitRevision)
                 .Select(i => new GitItemStatusWithParent(i.Group.Tag<GitRevision>(), i.Tag<GitItemStatus>()));
 
+        public IEnumerable<GitItemStatus> FirstGroupItems
+        {
+            get
+            {
+                if (FileStatusListView.Groups.Count == 0)
+                {
+                    return new List<GitItemStatus>();
```

Comment:
```suggestion
                    return Enumerable.Empty<GitItemStatus>();
```

Reply:
Not supported
![image](https://user-images.githubusercontent.com/6248932/77847809-19e02180-71c0-11ea-9b95-5042ca2ab601.png)

Reply:
Try `Array.Empty<GitItemStatus>()`

Reply:
Using yield below required use here too: yield brake